src/Parsers/ASTCreateFunctionQuery.cpp

```diff
@@ -81,6 +81,7 @@ void ASTCreateFunctionQuery::formatImpl(const IAST::FormatSettings & settings, I
             settings.ostr << fmt::format("AUTH_HEADER '{}'\n", function_core->children[1]->as<ASTLiteral>()->value.safeGet<String>());
             settings.ostr << fmt::format("AUTH_KEY '{}'\n", function_core->children[2]->as<ASTLiteral>()->value.safeGet<String>());
         }
+        settings.ostr << fmt::format("EXECUTION_TIMEOUT {}", function_core->children.back()->as<ASTLiteral>()->value.safeGet<UInt64>());
```

Comment:
What if the user does not set 'command_execution_timeout'?

Reply:
the default value 10000 will be inserted and It is ensured that children of function_core is always at least 2( the first one is always auth_method and the last one is always execution timeout )